install-worker.sh

```diff
@@ -29,6 +29,15 @@ sudo yum install -y \
 # Make sure Amazon Time Sync Service starts on boot.
 sudo chkconfig chronyd on
 
+# Make sure that chronyd syncs RTC clock to the kernel.
+cat <<EOF | sudo tee -a /etc/chrony.conf
+
```

Comment:
nit: can you remove the trailing and leading empty newlines?

Reply:
Updated